### DISCUSSION TITLE
Keep every done date&time property

### DISCUSSION THREAD

**Discussion by yoghurtdewdew** [YU000jp/logseq-plugin-confirmation-done-task#9] [Ideas]
When I click on checkbox to cancel done and then redo done, property added will not update. I would prefer to add another new date&time property while keep the previous done date&time property, so that I can see different completion dates&time properies for review them.

**Comment by YU000jp** [comment]
Certainly, that approach can be considered as one workflow. However, it's challenging to incorporate multiple states in its current form. I believe we need a different mindset, but at this stage, I can't think of a clever solution. I'll keep it in mind, though, as it might be achievable someday.

**Comment by YU000jp** [comment]
https://github.com/YU000jp/logseq-plugin-confirmation-done-task/issues/22
